### ISSUE TITLE
[Feature Request]: New File Manager

### ISSUE THREAD

**Issue by jenil1122** [prasanthrangan/hyprdots#1558]
Hi , recently i have discovered a super cool file manager. 
[Link](https://github.com/yorukot/superfile/blob/main/README.md)
I wonder if it can be added with a specific theme or replace dolphin..

**Comment by rubiin** [comment]
I think replacing dolphin will be a breaking change . Maybe an option to add it as extras. You can request on hyde-cli

**Comment by kRHYME7** [comment]
>  I wonder if it can be added with a specific theme or replace dolphin..

I tried it, but it won't follow the terminal colors. 

Can you try to check their docs to see how to handle theming on spf?

**Comment by Itz-Abhishek-Tiwari** [comment]
![image](https://github.com/prasanthrangan/hyprdots/assets/69257109/9f3ce8a4-9682-4691-9d40-15b0cf067bfd)

**Comment by abenezerw** [comment]
How did you end up solving this issue @Itz-Abhishek-Tiwari

**Comment by rubiin** [comment]
I like yazi :D changed from ranger

**Comment by abenezerw** [comment]
Yup yazi is way faster as it’s rust compared to go with spf. Anyways we can close out this pr as they are both terminal based file managers and swapping out dolphin would be a major change. @jenil1122
